front/src/modules/users/components/Avatar.tsx

```diff
@@ -6,10 +6,11 @@ import { stringToHslColor } from '~/utils/string-to-hsl';
 import { getImageAbsoluteURIOrBase64 } from '../utils/getProfilePictureAbsoluteURI';
 
 export type AvatarType = 'squared' | 'rounded';
+export type AvatarSize = 'xl' | 'lg' | 'md' | 'sm' | 'xs';
```

Comment:
it works but we prefer using enums. Not blocking merged as we still have a few type like this in the codebase